### PR TITLE
Fix #1153

### LINKED ISSUE #1153: Bot can't move in y=256 or 255
[X] The [FAQ](https://github.com/PrismarineJS/mineflayer/blob/master/docs/FAQ.md) doesn't contain a resolution to my issue 

## Versions
 - mineflayer: ^2.23.0
 - server: paper 
 - node: 12 LTS

## Detailed description of a problem
Bot thinks that it is surrounded by blocks in y=256 or 255.

So it cannot move.

## Expected behavior
Bot should move normally

## Additional context
Works perfectly in other ground:
https://streamable.com/ud7kgn

Can't move in y=255 or 256
https://streamable.com/myfeqy

(sorry for the item names. I didn't name them.)

### PR DESCRIPTION
**Pull request by Karang**
Needs prismarine-chunk to be released and version to be bumped

### CODE REVIEW

**Comment by rom1504** [comment]
Can you do the prismarine-chunk pr for release ? On a phone, can't do it

**Comment by Karang** [comment]
> Can you do the prismarine-chunk pr for release ? On a phone, can't do it

There seems to have an other issue related to biomes in latest versions (since biomes are now 3D) we might want to fix that too in pchunks before releasing

**Comment by Karang** [comment]
also closes https://github.com/PrismarineJS/mineflayer/issues/1155

File: docs/history.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 2.24.0
+* Fix getBlockAt when outside bounds
+* Improve documentation and examples
+
```
Comment (rom1504) [inline docs/history.md:4]: can you add : 
* Add ability to change the skin parts of a bot (thanks @Naomi-alt )

Comment (Karang) [reply docs/history.md:5]: sure :) done

**Pull request merged**